### DISCUSSION TITLE
Require PHP version >=8.0.2

### DISCUSSION THREAD

**Discussion by Aharonyan** [getsentry/sentry-php#1284] [Q&A] [answered]
After installing via composer sentry/SDK I see (https://prnt.sc/26k5hvm) this issue but in your documentation, I can see that you supporting  PHP 7.4 too

Can you help me?

**Comment by stayallive** [comment, marked as answer]
This is a composer error and has nothing to do with us or our SDK because we indeed support PHP 7.4.

You have probably ran `composer require sentry/sdk` on a machine that has PHP 8 installed and are running that site under PHP 7.4 triggering the error.

Make sure you run composer under the correct PHP version and the issue should be fixed.

**Reply by Aharonyan** [reply]
Thanks mate! it is working now!
